functions/middleware.js

```diff
@@ -5,8 +5,8 @@ functions.config()
 exports.validateFirebaseIdToken = (req, res, next) => {
 
   if ((!req.headers.authorization || !req.headers.authorization.startsWith('Bearer ')) &&
-      !req.cookies.__session) {
-    res.status(403, {"error":"Token no valid"});

```

Comment:
the ``res.status(code, json)`` is a method that not exits so the express still waiting for the ``send()`` or the ``json()`` call